### DISCUSSION TITLE
2022  Databend T-shirt is Coming!

### DISCUSSION THREAD

**Discussion by PsiACE** [databendlabs/databend#5886] [Announcement]
> Databend presents a brand new gift for contributors! 

Summer is coming! 

Databend is actively moving forward with the work in [**Release proposal: Nightly v0.8**](https://github.com/datafuselabs/databend/issues/4591). And we will soon start planning for **Nightly v0.9**.

From today, contributors will receive a T-shirt for participation.

**T-shirt  Show**

![20220609-094510](https://user-images.githubusercontent.com/36896360/172997237-10a4d757-83a0-44c8-8f6a-bbc63c001d68.png)

**T-shirt Size**

Sizes S to 3XL are currently available. 

![t-size](https://user-images.githubusercontent.com/36896360/179904248-4c7daaf5-a3ef-4979-abc4-80dbcc198286.png)
![t-measure](https://user-images.githubusercontent.com/36896360/180033373-92aff8c5-2b7d-4b6c-a545-a563f07b252e.png)

**Note**

- The units of measure for the data in the table are **centimetres(cm)**.
- There may be inconsistencies of 1-3 cm due to different measuring tools and methods.

---

Please provide your subscription to [hi@datafuselabs.com](mailto:hi@datafuselabs.com) like this:

> GitHub Nickname:
Address:
Name:
Phone:
Size:
Source: Contributors #5886

**Comment by nickhuang1127** [comment]
Give a like

**Comment by warriorsfly** [comment]
Play up!

**Comment by bohutang** [comment]
Thanks List:
@jialugong
@AngleNet 
@doki23
@ygf11
@RinChanNOWWW
@Kikkon
@devillove084
@cadl
@wfxr 
@TennyZhuang
@ClSlaid
@cuichenli
@Veeupup 
@PragmaTwice
@Defined2014
@gaoxinge
@GrapeBaBa
@TianLangStudio
@aseaday
@pymongo
@e1ijah1
@IDJack
@eastfisher
@JasonLi-cn
@xychu
@junaire
@guzzit
@eliasyaoyc 
@TszKitLo40
@zzzdong

**Comment by Xuanwo** [comment]
Thank @wfxr for his contribution, feel free to apply for a Databend T-Shirt here!

**Reply by Xuanwo** [reply]
Also @TianLangStudio for your contribution at https://github.com/datafuselabs/databend/pull/5922

**Reply by wfxr** [reply]
Wow! Such a surprise!

**Comment by TianLangStudio** [comment]
It's a surprise Thanks

**Comment by lalawuu** [comment]
I think you could provide a dark mode one !
